readme.md

```
@@ -117,19 +117,19 @@ _config.multiLanguage_
 
 In this plugin:
 ```
-> yarn
-> yarn link
+> pnpm
+> pnpm pack
 ```
 
 In your project:
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)